searchcore/src/tests/proton/documentdb/lid_space_compaction/lid_space_compaction_test.cpp

```diff
@@ -91,16 +88,16 @@ struct MyHandler : public ILidSpaceCompactionHandler
     }
     virtual uint32_t getSubDbId() const override { return 2; }
     virtual LidUsageStats getLidStatus() const override {
-        ASSERT_TRUE(_handleMoveCnt < _stats.size());
+        assert(_handleMoveCnt < _stats.size());
```

Comment:
Please keep using ASSERT_TRUE

Reply:
The ASSERT_TRUE macro returns from the function, so that cannot be used here.